Cargo.toml

```diff
@@ -25,6 +25,10 @@ repository   = "https://github.com/0xPolygonMiden/miden-node"
 rust-version = "1.85"
 version      = "0.8.0"
 
+# Optimise the cryptography for faster tests involving account creation.
+[profile.test.package.miden-crypto]
+opt-level = 2
+
```

Comment:
I used 

```
cargo flamegraph miden-faucet --unit-test -- test_website
```

to identify the cause. I also tried several other `miden-xxx` and `winter-xxx` deps, but maybe there are some I missed, or there is some combination that should be used?

Reply:
The slow part is `SecretKey::with_rng`, right? On my machine it takes 51s on current next, while actual account creation takes 81ms, so I think promoting miden-crypto to a higher opt level is the right thing to do.